### DISCUSSION TITLE
How to configure how many days of data to display in graphs?

### DISCUSSION THREAD

**Discussion by logichammer** [NightscoutFoundation/xDrip#3383] [Q&A]
I feel like this is probably a pretty noob question given that we've been using xdrip for years and years, but is there a way to allow the graphs to display more data than 24 hours?

I've looked in the app and in the discussion forum here and can't readily find it. I'm sure it exists and I'm just being silly. Can someone point me in the right direction?

**Comment by Navid200** [comment]
Please explain what you mean.
You can access readings from previous days using History from the main menu.

But, you can only show up to 24 hours on the main screen.

**Reply by logichammer** [reply]
Dear Lord, when I said noob, I knew it was a noob question!

And if I want to add a calibration, I should probably click the add calibration link?  :)

Please do not ask me why I thought history was a history of the xdrip project as opposed to the history of my BGs! I was buried in the advanced optional xdrip features when it was just part of the main navigation. Many thanks for your help and for answering my silly question!
